docker/test/stateless/run.sh

```diff
@@ -3,6 +3,10 @@
 # fail on errors, verbose and export all env variables
 set -e -x -a
 
+service etcd start && sleep 5
+
+echo "Etcd"
```

Comment:
better make it optional for case when we want to test etcd?

Reply:
Of course, It is just for test here in PR.